### DISCUSSION TITLE
有关输入法皮肤的外观显示问题，恳请各位大佬赐教

### DISCUSSION THREAD

**Discussion by lovimnnqbi** [rime/home#1600] [Q&A] [answered]
我把小狼毫从0.15.0版本更新到0.16.1版本后发现小狼毫可以跟随系统切换亮、暗模式，但切换到暗色模式后输入法皮肤的显示会出现异常。由于我是小白用户，之前使用配置也是跟着大佬们在网上发布的教程以及网上分享的部署包里的注释进行的，没有能力找到这个问题的根源在哪，在网上也没有找到类似的情况，于是来此求各位大佬给条可以解决问题的明路，或者能教教我如何让小狼毫一直使用亮色模式也是可以的。先在此谢过各位大佬了！
**下面是我的使用环境：**
- 电脑系统：win10
- 小狼毫版本：0.16.1
- 小狼毫部署包：[汉心自然码双拼辅助码](http://yaoxiazai.ysepan.com/)
- 自动切换系统亮、暗模式的软件：[Auto Dark Mode](http://yaoxiazai.ysepan.com/)
------
**问题效果图：**
亮色模式：
![image](https://github.com/rime/home/assets/124759361/e2e7b884-7f56-4707-bc9e-d982e007ac6a)
暗色模式：
![image](https://github.com/rime/home/assets/124759361/c916358f-4930-4de0-af34-da8e56a91274)

**Comment by michael94941** [comment]
这个表情和表情符号是怎么跟上去的？要配置什么吗？可以分享一下吗？

**Reply by fxliang** [reply, marked as answer]
暗色的配色不完整导致的，你可以将暗色的配色方案和亮色的只改一个高亮候选背景色对比即可

**Reply by lovimnnqbi** [reply]
@michael94941 我也是个小白。我是用的汉心快打的部署包，可以直接在配置文件里面开启。你可以去网上搜搜“rime添加emoji输入方案”，会有一些教程，你试试有没有用。

**Reply by lovimnnqbi** [reply]
@fxliang 感谢大佬解惑！我研究了一下暗色模式的皮肤配色代码，确实有问题。现在问题已经解决了。
